Frontend/src/components/pages/Company.jsx

```diff
@@ -214,6 +271,87 @@ export default function Company() {
             Companies
           </Button>
 
+          <Box display="inline"
+              sx = {{
+                paddingBlock: 2,
+                float: 'right'
+              }}
+          >
+              <IconButton
```

Comment:
Zaboravio si dodati tooltip na buttone. Nadalje size treba biti small. 
- vrijedi za sve iconbuttone
Evo kako je to u Tablecomponent napravljeno:

<Tooltip title="Edit" key="Edit">
                    <IconButton
                      size="small"
                      onClick={() => handleEdit(result)}
                      sx={{
                        color: "white",
                        backgroundColor: "#1976d2",

                        borderRadius: 1,

                        width: { xs: 20, md: "unset" },
                      }}
                    >
                      <EditIcon />
                    </IconButton>
                  </Tooltip>